aspnetcore/security/authentication/certauth.md

```diff
@@ -183,13 +183,13 @@ In *Program.cs*, configure Kestrel as follows:
 ```csharp
 public static IWebHost BuildWebHost(string[] args) =>
```

Comment:
This will have to return `IHost` now instead of `IWebHost`.

Reply:
sec.. good catch. 
sec.

Reply:
fixed.